typings/test.ts

```diff
@@ -1,14 +1,14 @@
 // This is a test file for the TypeScript typings.
 // It is not intended to be used by external users.
-import Telegraf, { Markup } from './index';
+import Telegraf, { Markup, Middleware, ContextMessageUpdate } from './index';
 
 
 const randomPhoto = 'https://picsum.photos/200/300/?random'
-const sayYoMiddleware = ({ reply }, next) => reply('yo').then(() => next())
+const sayYoMiddleware: Middleware<ContextMessageUpdate> = ({ reply }, next) => reply('yo').then(() => next && next())
 
 const {reply} =  Telegraf;
 
-const bot = new Telegraf(process.env.BOT_TOKEN)
+const bot = new Telegraf(process.env.BOT_TOKEN || '')
```

Comment:
Fixes error TS2769: No overload matches this call: `process.env.BOT_TOKEN` can be `undefined`, while the constructor expects a string.